packaging/standalone/src/main/distribution/shell-scripts/bin/Neo4j-Management.psd1

```diff
@@ -21,9 +21,9 @@
 
 
 @{
-ModuleVersion = '2.3.0'
+ModuleVersion = '3.0.0'
 
-GUID = 'dd9cad34-ad03-439b-b347-590625302c38'
+GUID = '2a3e34b4-5564-488e-aaf6-f2cba3f7f05d'
```

Comment:
While you can't import a module by GUID, it looks like that's the intention, where the GUID is the primary key for identifying a module.  Also, you could conceivably have the same public facing functions etc. (think public api contract) which would have the same version but a different GUID because the private implementation is different.  A bit like the last number in Semver.  Again, ultimately, it doesn't really affect us as this module isn't published as such.  I can keep it the same if required.